packages/ui/src/components/templates/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,153 @@
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {Dimensions, StyleSheet, View} from 'react-native';
+import {
+  BarChartDashboard,
+  LineChartDashboard,
+  PieChartDashboard,
+} from '../../molecules';
+import {ScrollView} from '../../atoms';
+
+interface Data {
+  value: number;
+  color?: string;
+  label: string;
+}
+
+interface Graph {
+  type: 'pie' | 'bar' | 'line';
+  dataList: Data[][];
+  title?: string;
+}
+
+interface Line {
+  graphList: Graph[];
+}
+
+interface DashboardProps {
+  style?: any;
+  lineList: Line[];
+}
+
+const styleGraph = (nbGraphInLine: number, type: 'style' | 'width') => {
+  let style = null;
+  let width = null;
+
+  if (nbGraphInLine === 1) {
+    style = {width: Dimensions.get('window').width - 60};
+    width = Dimensions.get('window').width - 160;
+  } else if (Dimensions.get('window').width < 500 && nbGraphInLine !== 1) {
+    style = {width: Dimensions.get('window').width / 2 - 15};
+    width = Dimensions.get('window').width / 4;
+  } else if (nbGraphInLine === 2) {
+    style = {width: Dimensions.get('window').width / 2 - 60};
+    width = Dimensions.get('window').width / 3 - 60;
+  } else if (nbGraphInLine === 3) {
+    style = {width: Dimensions.get('window').width / 3 - 60};
+    width = Dimensions.get('window').width / 5 - 60;
+  } else if (nbGraphInLine === 4) {
+    style = {width: Dimensions.get('window').width / 4 - 20};
+    width = Dimensions.get('window').width / 6 - 60;
+  }
+
+  return type === 'style' ? style : width;
+};
+
+const LineChartDashboardRender = (datasets, key, nbGraphInLine, title) => {
+  return (
+    <LineChartDashboard
+      datasets={datasets}
+      key={key}
+      style={styleGraph(nbGraphInLine, 'style')}
+      widthGraph={styleGraph(nbGraphInLine, 'width')}
+      title={title}
+    />
+  );
+};
+
+const PieChartRender = (datasets, key) => {
+  return <PieChartDashboard datasets={datasets} key={key} legend={true} />;
+};
+
+const BarChartDashboardRender = (datasets, key, nbGraphInLine, title) => {
+  return (
+    <BarChartDashboard
+      datasets={datasets}
+      key={key}
+      style={styleGraph(nbGraphInLine, 'style')}
+      widthGraph={styleGraph(nbGraphInLine, 'width')}
+      title={title}
+    />
+  );
+};
+
+const Dashboard = ({style, lineList}: DashboardProps) => {
+  return (
+    <ScrollView style={[styles.container, style]}>
+      {lineList?.map((line, indexLine) => {
+        const nbGraphInLine =
+          line.graphList?.length > 4 ? 4 : line.graphList?.length;
```

Comment:
"4" should be define in a variable to make it understandable that it's a mac number of graph for a line 
I think it should got to the same helper file as the Data interface.